2019/python/rules_engine.py

```diff
@@ -72,7 +72,7 @@ def results_wrapper(self, fail_df=None, field_name=None, edit_name=None, row_typ
 				failed_rows = list(fail_df.uli)
 				self.update_results(edit_name=edit_name, edit_field_results=result, row_type=row_type, fields=field_name, row_ids=failed_rows, fail_count=count)
 			else:
-				self.update_results(edit_name=edit_name, edit_field_results=result, row_type=row_type, fields=field_name)
+				self.update_results(edit_name=edit_name, edit_field_results=result, row_type=row_type, fields=field_name, row_ids='TS', fail_count=1)
```

Comment:
Can you add a comment explaining why the TS fail count was hardcoded to 1?

Reply:
Sure!